devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/examples/resteasy-jackson-example/java/src/test/java/org/acme/rest/json/NativeFruitResourceIT.java

```diff
@@ -0,0 +1,9 @@
+package org.acme.rest.json;
```

Comment:
You need to move the native test to `src/native-test/java/org/acme/rest/json/NativeFruitResourceIT.java`, this is a `gradle` specificity (for maven the file will be automatically moved to the test directory), maybe we need to add this to the doc..